src/unixport/detex/decompress-bc.c

```diff
@@ -0,0 +1,243 @@
+/*
+
```

Comment:
Used detex for low level decomp since GLI was poorly documented/buggy and libpng seemed not to support the relevant decomp algos